cluster/gce/trusty/node.yaml

```diff
@@ -162,8 +163,16 @@ start on stopped kube-env
 
 script
 	. /etc/kube-env
-	# If kubelet or kube-proxy is not installed in the image, pull release binaries and put them in /usr/bin.
-	if ! which kubelet > /dev/null || ! which kube-proxy > /dev/null; then
+	# If both kubelet and kube-proxy test args are set, this is a testing cluster.
+	# We pull binaries and put them in /usr/local/bin. If it is not a testing
```

Comment:
For ubuntu, this is true. But for the customized GCE images, we cannot avoid it, because kubernetes binaries are installed in /usr/bin. For those images, if we run cluster with built-in binaries, they are in /usr/bin. But for continuous test, binaries cannot be put in non-writable /usr/bin. So, we have to use a different dir.